yb_os_maint/yb_os_maint.py

```diff
@@ -364,6 +479,38 @@ def stop_node(api_host, customer_uuid, universe, api_token, node):
         else:
             log('No x-cluster replications were found to pause')
 
+        ## Step down if master
```

Comment:
This block repeats the SAME API call, and should be simplified to something like:
`
def Get_Master_Leader:
   <<API CALL>>
retries = 10
ldr=Get_Master_Leader()
while retries-- > 0  and  ldr['privateIP' == node['cloudInfo']['privateIP']
    <Stepdown>
    Get_Master_leader()

If retries <=0:
   <Complain in log>>
`